echo_handlers.go

```diff
@@ -19,11 +19,11 @@ import (
 
 func (svc *Service) ValidateUserMiddleware(next echo.HandlerFunc) echo.HandlerFunc {
 	return func(c echo.Context) error {
-		// TODO: check if login is required and check if user is logged in
-		//sess, _ := session.Get(CookieName, c)
-		// if user == nil {
-		// 	return c.NoContent(http.StatusUnauthorized)
-		// }
+		if svc.lnClient == nil {
```

Comment:
I think this is a good check but doesn't fit the name of the middleware, and is not related to the middleware that this would be for (checking if the user has unlocked NWC / has authed in http mode)

Reply:
Changed to `ValidateNodeMiddleware`